packages/gitgraph-core/src/commit.ts

```diff
@@ -0,0 +1,174 @@
+import { CommitStyle } from "./template";
+import Branch from "./branch";
+
+export interface CommitOptions {
+  author: string;
+  subject: string;
+  style: CommitStyle;
+  body?: string;
+  notes?: string;
+  refs?: string[];
```

Comment:
It makes me wonder about tags. Technically, I understand tags as refs. Branches also are refs. And `HEAD` is another kind of refs. 

But I wouldn't make a difference between the list of `refs` (currently, branch names & HEAD) and `tags`. 

What do you think about merging them?

Though, I agree to expose a different `.tag()` command. However, a `refs` options doesn't make sense to me (it's technical details, shouldn't be part of the public API).

Reply:
We need a way to show (or not) the refs on the graph, it's why this `refs` is public, for me it's interesting to have different style between `refs` and `branches` and I think that it was the original design for this.

Also remember here that we don't manipulate the `commit` class directly, this class is always instanciate by a `branch` ;) (just to have a little bit more context)

Reply:
I'm fine keeping `refs` and `tags` separated for now as I don't have concrete issue because of that 👍